forge-gui/res/cardsfolder/d/dire_wolves.txt

```diff
@@ -2,7 +2,6 @@ Name:Dire Wolves
 ManaCost:2 G
 Types:Creature Wolf
 PT:2/2
-S:Mode$ Continuous | Affected$ Card.Self | AddKeyword$ Banding | CheckSVar$ CheckPLAINS | SVarCompare$ GE1 | Description$ CARDNAME has banding as long as you control a Plains.
-SVar:CheckPLAINS:Count$Valid Plains.YouCtrl
+S:Mode$ Continuous | Affected$ Card.Self | AddKeyword$ Banding | IsPresent$ Plains.YouCtrl | Description$ CARDNAME has banding as long as you control a Plains.
 AI:RemoveDeck:All
```

Comment:
Is this RemoveDeck:All because of Banding?
Maybe still add `SVar:BuffedBy:Plains` ?